lasagne/layers/__init__.py

```diff
@@ -8,4 +8,5 @@
 from .shape import *
 from .merge import *
 from .normalization import *
+from .prelu import *
```

Comment:
That doesn't make too much sense imo, the full 'path' would be `lasagne.layers.elemwise`, no need to tack another `layer` at the end :) Also it doesn't make sense to call the module `elemwiselayer` if we then proceed to put multiple `Layer` subclasses in it.